perma-payments/perma_payments/tests/test_models.py

```diff
@@ -666,6 +746,93 @@ def test_response_save_new_with_encrypted_full_response_ur(mocker, barebones_upd
     encrypted.assert_called_once_with(mocker.sentinel.stringified)
 
 
+@pytest.mark.django_db
+def test_response_save_new_with_encrypted_full_response_pr(mocker, purchase_request, spoof_django_post_object):
+    # mocks
+    stringified = mocker.patch('perma_payments.models.stringify_data', return_value=mocker.sentinel.stringified)
+    encrypted = mocker.patch('perma_payments.models.encrypt_for_storage', return_value=b'someencryptedbytes')
+
+    # call
+    fields = {
+        'related_request': purchase_request,
+        'decision': random.choice([choice[0] for choice in Response._meta.get_field('decision').choices]),
+        'reason_code': SENTINEL['reason_code'],
+        'message': SENTINEL['message']
+    }
+    Response.save_new_with_encrypted_full_response(PurchaseRequestResponse, spoof_django_post_object, fields)
+    response = purchase_request.purchase_request_response
+
+    # save worked
+    assert isinstance(response, PurchaseRequestResponse)
+    for key, value in fields.items():
+        assert getattr(response, key) == value
+    assert response.full_response == b'someencryptedbytes'
+
+    # mocks called as expected
+    stringified.assert_called_once_with(spoof_django_post_object)
+    encrypted.assert_called_once_with(mocker.sentinel.stringified)
+
+
+# PurchaseRequestResponse
+
+def test_prr_inherits_from_outgiong_transaction():
```

Comment:
Typo in function name here, doubt it matters

Reply:
thanks